www/js/init.js

```diff
@@ -26,7 +26,7 @@ require.config({
     baseUrl: 'js/lib',
```

Comment:
It's probably not the cause of our issues, but loading the fontawesome javascript files should be done by requirejs too. If you remove the `script` tags in index.html and apply the suggestion below, it should be enough.

I did not find any reference of asynchronous things done by bootstrap, so I suspect it could come from the way we configure requirejs.
This piece of code makes sure bootstrap is loaded before running app.js (in the sense of requirejs). I think it was not the case before, and requirejs was loading bootstrap and app.js in parallel.
Could you try this (without the CSS workaround)?

```suggestion
    baseUrl: 'js/lib',
    paths: {
        'jquery': 'jquery-3.2.1.slim',
        'bootstrap': 'bootstrap.bundle',
        'fontawesome': 'fontawesome/fontawesome',
        'fontawesome-solid': 'fontawesome/solid'
    },
    shim: {
        'jquery' : {
            exports : '$'
        },
        'bootstrap': {
            deps: ['jquery', 'fontawesome', 'fontawesome-solid']
        }
    }
});

requirejs(['bootstrap'], function (bootstrap) {
    requirejs(['../app']);
});

```

Reply:
@mossroy Thank you! This code seems to solve the async loading issue, and means that the bootstrap functions are ready and available for use as soon as app.js runs. I've removed all workarounds and I don't see the error appearing on quick tests.

However, as you can see, Travis is still failing with the same error on Chrome 58... 😒